FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - [fixed] Fixed database creation on tvOS. (#6612)
 - [changed] Updated public API documentation to no longer reference removed APIs. (#6641)
 - [fixed] Updated `activateWithCompletion:` to use completion handler for experiment updates. (#3687)
+- [changed] Add support for other Firebase products to integrate with Remote Config. (#6692)
```

Comment:
Should move to a 7.1.0 section.  7.0.0 is past code freeze.

Reply:
Done.